corehq/apps/hqcase/management/commands/ptop_fast_reindex_xforms.py

```diff
@@ -17,11 +17,30 @@ class Command(ElasticReindexer):
 
     sort_key_include_docs = True
 
-    def get_extra_view_kwargs(self):
-        return {
-            'startkey': ['XFormInstance'],
-            'endkey': ['XFormInstance', {}],
-        }
+    def full_couch_view_iter(self):
+        # copied from couchforms/_design/filters/xforms.js
+        doc_types = [
+            'XFormInstance',
+            'XFormArchived',
+            'XFormError',
+            'XFormDeprecated',
+            'XFormDuplicate',
+            'XFormInstance-Deleted',
+            'HQSubmission',
+            'SubmissionErrorLog',
```

Comment:
@sravfeyn these are all the doc types that are pulled in by realtime pillowtop, so I'm making the fast reindexing do the same thing